extensions/emmet/src/selectItemHTML.ts

```diff
@@ -19,7 +19,7 @@ export function nextItemHTML(document: vscode.TextDocument, selectionStart: vsco
 	if (currentNode.type !== 'comment') {
 		// If cursor is in the tag name, select tag
 		if (currentNode.open &&
-			selectionEndOffset < currentNode.open.start + currentNode.name.length) {
+			selectionEndOffset <= currentNode.open.start + currentNode.name.length) {
```

Comment:
Can you give an example where this change is needed?

Reply:
Consider this with your cursor between **i** and **v** in the very first line 
`<div>
      <span></span>
</div>
`

It should select the `<div>` tag but it selects the child `<span>`. This is because the condition changed in the code does not consider this cursor position as part of the current node.